commits 03/2019

MarcelAnderson opened the pull request.

MarcelAnderson committed:
Update x1131_memoryobject1.stringtable

double word (am)

MarcelAnderson committed:
Update x1131_memoryobject1.stringtable

komma - Nebensatz

MarcelAnderson committed:
Update x1131_memoryobject1.stringtable

deleted whitespace

MarcelAnderson committed:
Update x1131_memoryobject2.stringtable

tiefergehend --> tiefer gehend

https://www.duden.de/sprachwissen/newsletter/Duden-Newsletter-100812

MarcelAnderson committed:
Update x1131_memoryobject2.stringtable

zurück zu erlangen --> zurückzuerlangen

MarcelAnderson committed:
Update x1131_memoryobject2.stringtable

not translated

MarcelAnderson committed:
Update x1131_memoryobject2.stringtable

not translated

MarcelAnderson committed:
Update x1131_memoryobject2.stringtable

Rechtschreibfehler

MarcelAnderson committed:
Update x1131_memoryobject3.stringtable

herrankommen --> herankommen

MarcelAnderson committed:
Update x1131_memoryobject3.stringtable

hinwegtauchen --> hinweg tauchen

MarcelAnderson committed:
Update x1131_memoryobject3.stringtable

halben --> halbem

MarcelAnderson committed:
Update x1131_memoryobject3.stringtable

not translated

MarcelAnderson committed:
Update x1131_reflections.stringtable

double word

MarcelAnderson committed:
Update x1131_reflections.stringtable

Standhaften --> standhaften

MarcelAnderson committed:
Update x1131_reflections.stringtable

neunzehnseitige --> neunzehn seitige

MarcelAnderson committed:
Update x1131_reflections.stringtable

Komma

MarcelAnderson committed:
Update x1131_reflections.stringtable

Komma

MarcelAnderson committed:
Update x1131_reflections.stringtable

not translated

MarcelAnderson committed:
Update x1131_reflections.stringtable

Komma

MarcelAnderson committed:
Update x1131_reflections.stringtable

translated

MarcelAnderson committed:
Update x1131_reflections.stringtable

not translated

MarcelAnderson committed:
Update x1131_reflections.stringtable

not translated

MarcelAnderson committed:
Update x1131_specter_barks.stringtable

reparierst --> repariere

MarcelAnderson committed:
Update x1151_calm_ghostlywoman.stringtable

Geisterhafte --> geisterhafte

MarcelAnderson committed:
Update x1151_calm_ghostlywoman.stringtable

Fünften --> fünften

MarcelAnderson committed:
Update x1151_calm_ghostlywoman.stringtable

whitespace

MarcelAnderson committed:
Update x1151_calm_ghostlywoman.stringtable

double "Nicht nicht "

MarcelAnderson committed:
Update x1151_calm_ghostlywoman.stringtable

hierhergebracht --> hierher gebracht

MarcelAnderson committed:
Update x1151_calm_ghostlywoman.stringtable

letze --> letzte

MarcelAnderson committed:
Update x1151_calm_ghostlywoman.stringtable

Lebwohl --> Lebewohl

MarcelAnderson committed:
Update x1151_calm_ghostlywoman.stringtable

deleted "mit"

MarcelAnderson committed:
Update x1151_calm_ghostlywoman.stringtable

hierhergebracht --> hierher gebracht

MarcelAnderson committed:
Update x1151_calm_ghostlywoman.stringtable

par --> paar

MarcelAnderson committed:
Update x1151_calm_ghostlywoman.stringtable

Leid --> leid

MarcelAnderson committed:
Update x1151_calm_ghostlywoman.stringtable

Missing "

MarcelAnderson committed:
Update x1151_calm_specter.stringtable

Whitespace deleted

MarcelAnderson committed:
Update x1151_datasphere_link.stringtable

Komma

MarcelAnderson committed:
Update x1151_datasphere_link.stringtable

Whitespace deleted

MarcelAnderson committed:
Update x1511_erritis_reflection.stringtable

Komma

MarcelAnderson committed:
Update x1511_erritis_reflection.stringtable

Lebwohl --> Lebewohl

MarcelAnderson committed:
Update x1511_erritis_reflection.stringtable

Whitespace deleted

MarcelAnderson committed:
Update x1511_erritis_reflection.stringtable

bestimmung --> Bestimmung

MarcelAnderson committed:
Update x1511_erritis_reflection.stringtable

Großschreibung: hüter, tiere

MarcelAnderson committed:
Update x1511_erritis_reflection.stringtable

helden --> Helden

MarcelAnderson committed:
Update x1511_erritis_reflection.stringtable

voyeure --> Voyeure

MarcelAnderson committed:
Update x1511_erritis_reflection.stringtable

wort --> Wort

MarcelAnderson committed:
Update x1511_erritis_reflection.stringtable

all big, then all small, so okay from speaking...

MarcelAnderson committed:
Update x1511_lady_anshe.stringtable

deleted double word "wirklich"

MarcelAnderson committed:
Update x1511_lady_anshe.stringtable

hab --> habe

MarcelAnderson committed:
Update x1511_lady_anshe.stringtable

ander Mal --> anderes Mal

MarcelAnderson committed:
Update x1511_lady_anshe.stringtable

verzeirten --> verzierten

MarcelAnderson committed:
Update x1511_lady_anshe.stringtable

näherschreitest --> näher schreitest

MarcelAnderson committed:
Update x1511_lady_anshe.stringtable

Luxeriöser --> Luxusiöser

MarcelAnderson committed:
Update x1511_lady_anshe.stringtable

teueraussehende --> teuer aussehende

MarcelAnderson committed:
Update x1511_lady_anshe.stringtable

Recht --> recht

MarcelAnderson committed:
Update x1511_lady_anshe.stringtable

Funklen --> Funkeln

MarcelAnderson committed:
Update x1511_lady_anshe.stringtable

 ihren Kinn --> ihrem Kinn

MarcelAnderson committed:
Update x1511_reflection_artaglio.stringtable

Komma

MarcelAnderson committed:
Update x1511_reflection_artaglio.stringtable

Komma

MarcelAnderson committed:
Update x1511_reflection_artaglio.stringtable

ein --> in

MarcelAnderson committed:
Update x1511_reflection_artaglio.stringtable

lass --> lasse

MarcelAnderson committed:
Update x1511_reflection_artaglio.stringtable

seinen --> seine

MarcelAnderson committed:
Update x1511_reflection_artaglio.stringtable

seinen --> seine

MarcelAnderson committed:
Update x1511_reflection_artaglio.stringtable

genausogut --> genauso gut

MarcelAnderson committed:
Update x1511_reflection_artaglio.stringtable

Reflektionen --> Reflexionen

MarcelAnderson committed:
Update x1511_reflection_barks_3.stringtable

hierhergebracht --> hierher gebracht

MarcelAnderson committed:
Update x1511_reflection_choi.stringtable

se --> sie

MarcelAnderson committed:
Update x1511_reflection_choi.stringtable

added whitespace

MarcelAnderson committed:
Update x1511_reflection_choi.stringtable

Komma

MarcelAnderson committed:
Update x1511_reflection_choi.stringtable

Lebwohl --> Lebewohl

MarcelAnderson committed:
Update x1511_reflection_choi.stringtable

Komma

MarcelAnderson committed:
Update x1511_reflection_corpuscular_maw.stringtable

korpuskularen --> korpuskulären

MarcelAnderson committed:
Update x1511_reflection_corpuscular_maw.stringtable

korpuskularen --> korpuskulären

MarcelAnderson committed:
Update x1511_reflection_corpuscular_maw.stringtable

korpuskulare --> korpuskuläre

MarcelAnderson committed:
Update x1511_reflection_inifere.stringtable

Trotz dem Fehlen --> Trotz des Fehlens

MarcelAnderson committed:
Update x1511_reflection_inifere.stringtable

standhälst --> standhältst

MarcelAnderson committed:
Update x1511_reflection_inifere.stringtable

Whitespace

MarcelAnderson committed:
Update x1511_reflection_inifere.stringtable

zwielichte --> zwielichtige

MarcelAnderson committed:
Update x1511_reflection_inifere.stringtable

pflücken --> Pflücken

MarcelAnderson committed:
Update x1511_reflection_inifere.stringtable

picken --> Picken

MarcelAnderson committed:
Update x1511_reflection_inifere.stringtable

Lebwohl --> Lebewohl

MarcelAnderson committed:
Update x1511_reflection_inifere.stringtable

Keiner --> keiner

MarcelAnderson committed:
Update x1511_reflection_inifere.stringtable

Misstände --> Missstände

MarcelAnderson committed:
Update x1511_reflection_inifere.stringtable

scheint an die --> scheint an dir

MarcelAnderson committed:
Update x1511_reflection_inifere.stringtable

vorbeizustrecken --> vorbeizustreifen

MarcelAnderson committed:
Update x1511_reflection_inifere.stringtable

zerissenes --> zerrissenes

MarcelAnderson committed:
Update x1511_reflection_inifere.stringtable

zerklöftete --> zerklüftete

MarcelAnderson committed:
Update x1511_reflection_inifere.stringtable

erlischen --> erlöschen

MarcelAnderson committed:
Update x1511_reflection_inifere.stringtable

genau so --> genauso

MarcelAnderson committed:
Update x1511_reflection_inifere.stringtable

zum lernen --> zum Lernen

MarcelAnderson committed:
Update x1511_reflection_nychthemeron.stringtable

deleted whitespace before "..."

MarcelAnderson committed:
Update x1511_reflection_nychthemeron.stringtable

Lebwohl --> Lebewohl

MarcelAnderson committed:
Update x1511_reflection_seria.stringtable

einem innerem Licht --> inneren

MarcelAnderson committed:
Update x1511_reflection_seria.stringtable

alles --> alles
(Wendung "das/mein/dein --> Ein und Alles werden großgeschrieben)

MarcelAnderson committed:
Update x1511_reflection_seria.stringtable

hierhergebracht --> hierher gebracht

MarcelAnderson committed:
Update x1511_reflection_seria.stringtable

Lebwohl --> Lebewohl

MarcelAnderson committed:
Update x1511_reflection_seria.stringtable

hineinstolpert --> hinein stolpert

MarcelAnderson committed:
Update x1511_reflection_seria.stringtable

wrong "?"

MarcelAnderson committed:
Update x1511_reflection_seria.stringtable

Konzentrier  --> Konzentriere

MarcelAnderson committed:
Update x1511_reflection_seria.stringtable

Gurglen --> Gurgeln

MarcelAnderson committed:
Update x1511_reflection_seria.stringtable

deleted whitespace before "..."

MarcelAnderson committed:
Update x1511_reflection_seria.stringtable

added whitespace after "..."

MarcelAnderson committed:
Update x1511_statue_tidalmere.stringtable

colors are written small

MarcelAnderson committed:
Update x1551_inifere_reflection.stringtable

Meine Leid --> Mein Leid

MarcelAnderson committed:
Update x2021_diviaticu_crisis.stringtable

dazupassenden --> dazu passenden

MarcelAnderson committed:
Update x2031_daughtersfathom_ghostlyman.stringtable

kannst --> kann

MarcelAnderson committed:
Update x2031_daughtersfathom_ghostlyman.stringtable

Lebwohl -- >Lebewohl

MarcelAnderson committed:
Update x2031_daughtersfathom_ghostlyman.stringtable

deleted whitespace

MarcelAnderson committed:
Update x2031_daughtersfathom_ghostlyman.stringtable

herbeibeschworen --> herbei beschworen

MarcelAnderson committed:
Update x2031_daughtersfathom_ghostlyman.stringtable

lass dich --> lasse dich

MarcelAnderson committed:
Update x2031_daughtersfathom_ghostlyman.stringtable

herausschmeißen --> rausschmeißen

MarcelAnderson committed:
Update x2031_daughtersfathom_ghostlyman.stringtable

deleted whitespace

MarcelAnderson committed:
Update x6051_specterfathom_endgame_corpse_si.stringtable

brilliantem --> brillantem

MarcelAnderson committed:
Update x6051_specterfathom_endgame_corpse_si.stringtable

nichs --> nichts

MarcelAnderson committed:
Update x6071_cnpcfathom_village_aligern.stringtable

metallernen --> metallenen

MarcelAnderson committed:
Update x6071_cnpcfathom_village_aligern.stringtable

scheien --> schreien

MarcelAnderson committed:
Update x6071_cnpcfathom_village_aligern.stringtable

sei --> sie

MarcelAnderson committed:
Update x6071_cnpcfathom_village_aligern.stringtable

sind --> ist

MarcelAnderson committed:
Update x6071_cnpcfathom_village_aligern.stringtable

Dort --> dort

MarcelAnderson committed:
Update x6071_cnpcfathom_village_aligern.stringtable

Leid --> leid

MarcelAnderson committed:
Update x6071_cnpcfathom_village_aligern.stringtable

Komma wrong

MarcelAnderson committed:
Update x6071_cnpcfathom_village_hoodedstatue_si_backer_obelisk.stringtable

es es --> sie dir

MarcelAnderson committed:
Update x6071_cnpcfathom_village_hoodedstatue_si_backer_obelisk.stringtable

freiziehen --> frei ziehen

MarcelAnderson committed:
Update x6071_cnpcfathom_village_hoodedstatue_si_backer_obelisk.stringtable

gedankvoll --> gedankenvoll

MarcelAnderson committed:
Update x6071_cnpcfathom_village_hoodedstatue_si_backer_obelisk.stringtable

wiederhallt -> widerhallt

MarcelAnderson committed:
Update x6071_cnpcfathom_village_hoodedstatue_si_backer_obelisk.stringtable

frei zuzeihen --> frei z ziehen

MarcelAnderson committed:
Update x6071_cnpcfathom_village_matkina.stringtable

Erstenschlägt --> Ersten schlägt

MarcelAnderson committed:
Update x6071_cnpcfathom_village_matkina.stringtable

als nächstes --> als Nächstes

MarcelAnderson committed:
Update x6071_cnpcfathom_village_matkina.stringtable

focussieren --> fokussieren

MarcelAnderson committed:
Update x6071_cnpcfathom_village_matkina.stringtable

lebewohl --> Lebewohl

MarcelAnderson committed:
Update x6071_cnpcfathom_village_matkina.stringtable

scheit --> schreit

MarcelAnderson committed:
Update x6071_cnpcfathom_village_matkina.stringtable

schreckt --> schreckst

MarcelAnderson committed:
Update x6071_cnpcfathom_village_matkina.stringtable

genung --> genug

MarcelAnderson committed:
Update x6071_cnpcfathom_village_matkina.stringtable

dacon --> davon

MarcelAnderson committed:
Update x6071_cnpcfathom_village_matkina.stringtable

heran zu kommen. --> heranzukommen.

MarcelAnderson committed:
Update x6071_cnpcfathom_village_matkina.stringtable

wrong Komma

MarcelAnderson committed:
Update x6071_tash.stringtable

kämpf --> kämpft

MarcelAnderson committed:
Update x6071_tash.stringtable

Verbenne --> Verbrenne

MarcelAnderson committed:
Update x6071_tash.stringtable

Verräte --> Verrate

MarcelAnderson committed:
Update x6071_tash.stringtable

Komma

MarcelAnderson committed:
Update x6071_tash.stringtable

weißglühendes --> weiß glühendes

MarcelAnderson committed:
Update x6071_tash.stringtable

licht --> Licht

MarcelAnderson committed:
Update x2021_trainerfathom_kamose_villon_diviaticu.stringtable

 ganz in schwarz gekleidet --> Schwarz, Substantiv

MarcelAnderson committed:
Update x2021_trainerfathom_kamose_villon_diviaticu.stringtable

Komma

MarcelAnderson committed:
Update x2021_trainerfathom_kamose_villon_diviaticu.stringtable

Komma

MarcelAnderson committed:
Update x2021_trainerfathom_kamose_villon_diviaticu.stringtable

Komma

MarcelAnderson committed:
Update x2021_trainerfathom_kamose_villon_diviaticu.stringtable

Komma

MarcelAnderson committed:
Update x2021_trainerfathom_kamose_villon_diviaticu.stringtable

seh mich --> sehe mich

MarcelAnderson committed:
Update x2021_trainerfathom_kamose_villon_diviaticu.stringtable

Whitespace

MarcelAnderson committed:
Update x2021_trainerfathom_kamose_villon_diviaticu.stringtable

dein *Innerstes* sehen --> innerstes

MarcelAnderson committed:
Update x2021_trainerfathom_kamose_villon_diviaticu.stringtable

Komma

MarcelAnderson committed:
Update x2021_trainerfathom_kamose_villon_diviaticu.stringtable

Konzentrier --> Konzentriere

MarcelAnderson committed:
Update x2021_trainerfathom_kamose_villon_diviaticu.stringtable

Zuhauseseins --> Zuhause seins

MarcelAnderson committed:
Update x2021_trainerfathom_kamose_villon_diviaticu.stringtable

Sille --> Stille

MarcelAnderson committed:
Update x2021_trainerfathom_kamose_villon_diviaticu.stringtable

starkaussehenden --> stark aussehenden

MarcelAnderson committed:
Update x2021_trainerfathom_kamose_villon_diviaticu.stringtable

Komma

MarcelAnderson committed:
Update x2031_daughtersfathom_ghostlywoman.stringtable

Vergrabenen --> vergrabenen

MarcelAnderson committed:
Update x2031_daughtersfathom_ghostlywoman.stringtable

Lebwohl --> Lebewohl

MarcelAnderson committed:
Update x2031_daughtersfathom_ghostlywoman.stringtable

Komma

MarcelAnderson committed:
Update x2031_daughtersfathom_ghostlywoman.stringtable

Komma

MarcelAnderson committed:
Update x2031_daughtersfathom_ghostlywoman.stringtable

Vergrabenen --> vergrabenen

MarcelAnderson committed:
Update x2031_daughtersfathom_ghostlywoman.stringtable

Selbst --> selbst

MarcelAnderson committed:
Update x2031_daughtersfathom_ghostlywoman.stringtable

Questionmark

MarcelAnderson committed:
Update x2031_daughtersfathom_ghostlywoman.stringtable

div

MarcelAnderson committed:
Merge pull request #3 from AurelioSilver/master

sync with master

MarcelAnderson committed:
schein --> scheint

MarcelAnderson committed:
hastiggemalten --> hastig gemalten

MarcelAnderson committed:
Komma

MarcelAnderson committed:
gehont --> geschliffen (bad translation)

MarcelAnderson committed:
gehont --> geschärft (bad translation)

MarcelAnderson committed:
Komma

MarcelAnderson committed:
Weltzerbrechend --> Welt zerbrechend

MarcelAnderson committed:
Komma

MarcelAnderson committed:
der Singenden Dornen --> der singenden Dornen

MarcelAnderson committed:
freizerren --> frei zerren

MarcelAnderson committed:
der geisterhafte Frau --> der geisterhaften Frau

MarcelAnderson committed:
Komma

MarcelAnderson committed:
essentielle --> essenzielle

MarcelAnderson committed:
Prophezeihung --> Prophezeiung

MarcelAnderson committed:
Neuer Satzbau

MarcelAnderson committed:
Komma

MarcelAnderson committed:
Reflektion --> Reflexion

MarcelAnderson committed:
hastiggebauten --> hastig gebauten

MarcelAnderson committed:
Gesichtsaudruck --> Gesichtsausdruck

MarcelAnderson committed:
geötetet --> getötet

MarcelAnderson committed:
ist --> sind

MarcelAnderson committed:
komt--> kommt

MarcelAnderson committed:
undreißt --> und reißt

MarcelAnderson committed:
hinabstreckt --> hinab streckt

MarcelAnderson committed:
Schließ --> Schließe

MarcelAnderson committed:
unnatürlicherweise --> unnatürlicher weise

MarcelAnderson committed:
facettiert --> fassettiert

MarcelAnderson committed:
Sptitze --> Spritze

MarcelAnderson committed:
sicht --> Sicht

MarcelAnderson committed:
as --> aus

MarcelAnderson committed:
verdunklen --> verdunkeln

MarcelAnderson committed:
Beide --> beide

MarcelAnderson committed:
ihem --> ihrem

MarcelAnderson committed:
af -- >auf

MarcelAnderson committed:
zurdrückt --> zerdrückt

MarcelAnderson committed:
blendem --> blendendem

MarcelAnderson committed:
des Weinen --> des Weinens

MarcelAnderson committed:
unsichtbate --> unsichtbare

MarcelAnderson committed:
verliert --> verlierst

MarcelAnderson committed:
halbgefangenes --> halb gefangenes

MarcelAnderson committed:
unterwasserliegenden --> unter Wasser liegenden

MarcelAnderson committed:
Komma

MarcelAnderson committed:
Komma

MarcelAnderson committed:
entzeiht --> entzieht

MarcelAnderson committed:
frage --> Frage

MarcelAnderson committed:
Komma

MarcelAnderson committed:
vielgezackt --> viel gezackt

MarcelAnderson committed:
höhlenähnlich --> höhlen ähnlich

MarcelAnderson committed:
Komma / Punkt

MarcelAnderson committed:
Leezeichen

MarcelAnderson committed:
vershcließt --> verschließt

MarcelAnderson committed:
Komma

MarcelAnderson committed:
keum --> kaum

MarcelAnderson committed:
shauen --> schauen

MarcelAnderson committed:
Komma

MarcelAnderson committed:
das --> dass

MarcelAnderson committed:
daren -- >daran

MarcelAnderson committed:
Komma

MarcelAnderson committed:
Komma

MarcelAnderson committed:
Unsterbilcher --> Unsterblicher

MarcelAnderson committed:
ewigwährend --> ewig während

MarcelAnderson committed:
Du hast Recht --> "Du hast recht

MarcelAnderson committed:
"Ich weiß, dass du Recht hast" --> "Ich weiß, dass du recht hast",

MarcelAnderson committed:
erlischen --> erlöschen

MarcelAnderson committed:
erlischen --> erlöschen

MarcelAnderson committed:
Komma

MarcelAnderson committed:
Komma, Whitespace

MarcelAnderson committed:
ersmal --> erstmal

MarcelAnderson committed:
Lanyrinths --> Labyrinths

MarcelAnderson committed:
Komma

MarcelAnderson committed:
Grammatik

MarcelAnderson committed:
Recht --> recht

MarcelAnderson committed:
Menschhiet --> Menschheit

MarcelAnderson committed:
Das --> Der

MarcelAnderson committed:
Wiedererkannbarem --> Wiedererkennbarem

MarcelAnderson committed:
begracht --> gebracht

MarcelAnderson committed:
empfielst --> empfiehlst

MarcelAnderson committed:
vegangen --> vergangen

MarcelAnderson committed:
Verlorenen See --> verlorenen See

MarcelAnderson committed:
typo

MarcelAnderson committed:
Komma

MarcelAnderson committed:
missing "

AurelioSilver commented on text/conversations/x_labyrinth/x6081_cnpcfathom_oasis_callistege.stringtable:
```diff
@@ -1105,7 +1105,7 @@ Plötzlich und gleichzeitig erlöschen ihre Echos, was sie allein mitten in eine
       <DefaultText>Callisteges Hand erscheint auf dem Glas und wird dann schnell weggeschnappt. Ein knirschender, metallischer Missklang steigt zu einem Crescendo, was die Wände des Labyrinths und die Glastür der Höhle schüttelt, bis er schließlich abklingt.
 
-Die Glastür schiebt sich offen und einer tiefschwarze Staub geißt wie Wein heraus.
+Die Glastür schiebt sich offen und tiefschwarzer Staub gießt wie Wein heraus.
```
man könnte auch sagen: schiebt sich auf

MarcelAnderson replied:
da hast du recht - hatte den ersten Satzteil wohl nicht bemerkt. soll ich ein neues commit schicken?

AurelioSilver replied:
Ja, bitte.

MarcelAnderson replied:
Anpassung erfolgt in commit:
https://github.com/AurelioSilver/tidesofnumenera-german-patch/pull/10/commits/a52322bca3e5e8f8bf0c2c305418622400b7f102

Passt so?

AurelioSilver replied:
Ja, danke.

AurelioSilver commented on text/conversations/x_labyrinth/x1131_memoryobject1.stringtable:
```diff
@@ -153,7 +153,7 @@ Du erlaubst dir ein kurzes Lächeln. Aber du weißt nur zu gut, dass in der Fern
     <Entry>
       <ID>24</ID>
-      <DefaultText>Walk away from the orb.</DefaultText>
+      <DefaultText>Geh weg von der Kugel.</DefaultText>
```
Geh von der Kugel weg.

MarcelAnderson replied:
Siehe https://github.com/AurelioSilver/tidesofnumenera-german-patch/pull/10/commits/432dd925a9e7f712a31609af884b8bdf02cb5757

AurelioSilver commented on text/conversations/x_labyrinth/x1131_memoryobject2.stringtable:
```diff
@@ -145,7 +144,7 @@ Und wieder ist da noch etwas. Deine Nackenhaare stellen sich auf. Etwas jenseits
     <Entry>
       <ID>23</ID>
-      <DefaultText>Walk away from the orb.</DefaultText>
+      <DefaultText>Geh weg von der Kugel.</DefaultText>
```
Geh von der Kugel weg.

MarcelAnderson replied:
Siehe https://github.com/AurelioSilver/tidesofnumenera-german-patch/pull/10/commits/432dd925a9e7f712a31609af884b8bdf02cb5757

AurelioSilver commented on text/conversations/x_labyrinth/x1131_memoryobject3.stringtable:
```diff
@@ -144,7 +144,7 @@ Ein Beben erschüttert den Boden unter deinen Füßen, als ob eine gewaltige Fau
     <Entry>
       <ID>23</ID>
-      <DefaultText>Walk away from the orb.</DefaultText>
+      <DefaultText>Geh weg von der Kugel.</DefaultText>
```
Geh von der Kugel weg.

MarcelAnderson replied:
Siehe https://github.com/AurelioSilver/tidesofnumenera-german-patch/pull/10/commits/432dd925a9e7f712a31609af884b8bdf02cb5757

AurelioSilver commented on text/conversations/x_labyrinth/x6071_cnpcfathom_village_matkina.stringtable:
```diff
@@ -285,12 +285,12 @@ Sie lacht. "Von mir aus."</DefaultText>
       <DefaultText>Du legst deine Hand auf ihre Schulter, um sie zu schubsen, und sie schlägt aus, schnell wie ein Blitz. Du ziehst gerade schnell genug deine Hand zurück.
 
-"Kommt schon, ihr Feiglinge!" scheit sie. "Kommt und kämpft! Kommt und *sterbt* für das, was ihr getan habt!"</DefaultText>
+"Kommt schon, ihr Feiglinge!" schreit sie. "Kommt und kämpft! Kommt und *sterbt* für das, was ihr getan habt!"</DefaultText>
```
Komma fehlt hier:  ", schreit sie

MarcelAnderson replied:
Siehe c7e688a

MarcelAnderson committed:
Changes for comment https://github.com/AurelioSilver/tidesofnumenera-german-patch/pull/10#discussion_r281031863

MarcelAnderson committed:
changes for conversation, see attached

https://github.com/AurelioSilver/tidesofnumenera-german-patch/pull/10#discussion_r281031929

https://github.com/AurelioSilver/tidesofnumenera-german-patch/pull/10#discussion_r281031952

https://github.com/AurelioSilver/tidesofnumenera-german-patch/pull/10#discussion_r281031968

MarcelAnderson committed:
conversation note https://github.com/AurelioSilver/tidesofnumenera-german-patch/pull/10#discussion_r281032644

AurelioSilver commented:
Prima, danke sehr!
